src/library-authoring/components/BaseCard.tsx

```diff
@@ -65,12 +65,23 @@ const BaseCard = ({
           title={
             <Icon src={itemIcon} className="library-item-header-icon" />
           }
-          actions={
-            // Wrap the actions in a div to prevent the card from being clicked when the actions are clicked
-            /* eslint-disable-next-line jsx-a11y/click-events-have-key-events,
-            jsx-a11y/no-static-element-interactions */
-            <div onClick={(e) => e.stopPropagation()}>{actions}</div>
-          }
+          actions={(
+            // eslint-disable-next-line jsx-a11y/click-events-have-key-events, jsx-a11y/no-static-element-interactions
+            <div
+              onClick={(e) => {
+                // Wrap the actions in a div to prevent the card from being clicked when the actions are clicked.
+                const target = e.target as HTMLElement;
+                const isDropdownToggle = target.closest('[data-toggle="dropdown"], .dropdown-toggle, [id*="dropdown"]');
+
+                // But allow dropdown coordination events to bubble up for proper dropdown behavior.
+                if (!isDropdownToggle) {
+                  e.stopPropagation();
+                }
+              }}
```

Comment:
@DanielVZ96 Could you apply the same to the section/subsection child preview cards:

![image](https://github.com/user-attachments/assets/4038a63d-e250-49c8-9d2e-2c6786d5692d)

And to the Unit child preview cards:

![image](https://github.com/user-attachments/assets/b02bcf3c-750f-42df-8e23-1debf1dc514e)